airflow/task/context/current.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@mik-laj what do you think about this localization? I'm wondering if it would be better to move this to `airflow/utils`

Reply:
I personally prefer a named folder over `utils` if possible.

Reply:
Hmm.. I feel this is wrong place for this functions. I agree "utils" isn't the best place but mostly things under "airflow/task" are for task_runners but  I don't know where we can put it.🤷‍♂️

Reply:
How about `operator/python.py`?  This will make import simple:
```python
from airflow.operators.python import task, get_current_context
```

Reply:
yeah that might work, since it only applies to functional dags

Reply:
new module ? `airflow/funcdag` ?

Reply:
@potiuk not sure about that because that would mean that we should move whole AIP-31 code there?